README.md

```diff
@@ -37,6 +37,20 @@ vars:
 
 For additional configurations for the source models, visit the [Facebook Ads source package](https://github.com/fivetran/dbt_facebook_ads_source).
 
+### Changing the Build Schema
+By default this package will build the Facebook Ads staging models within a schema titled (<target_schema> + `_stg_facebook_ads`) and the Facebook Creative History final models with a schema titled (<target_schema> + `_facebook_creative_history`) in your target database. If this is not where you would like your modeled Facebook data to be written to, add the following configuration to your `dbt_project.yml` file:
+
+```yml
+# dbt_project.yml
+
+...
+models:
+  facebook_creative_history:
```

Comment:
should this be `facebook_ads_creative_history`?
```suggestion
  facebook_ads_creative_history:
```

Reply:
It should thank you!